e2e-tests/insurance/cypress/e2e/journeys/policy/about-goods-or-services/validation/about-goods-or-services-validation.spec.js

```diff
@@ -65,14 +67,36 @@ context('Insurance - Policy - About goods or services page - form validation', (
       false,
     );
 
-    // final destination
+    // final destination known
     cy.submitAndAssertFieldErrors(
-      countryInput.field(FINAL_DESTINATION),
+      countryInput.field(FINAL_DESTINATION_KNOWN),
       null,
       1,
       expectedErrorsCount,
-      ABOUT_ERROR_MESSAGES[FINAL_DESTINATION].IS_EMPTY,
+      ABOUT_ERROR_MESSAGES[FINAL_DESTINATION_KNOWN].IS_EMPTY,
       false,
     );
   });
+
+  describe(`when ${FINAL_DESTINATION_KNOWN} is 'yes', but ${FINAL_DESTINATION} is not provided`, () => {
+    it(`should render a ${FINAL_DESTINATION} validation error`, () => {
+      cy.navigateToUrl(url);
+
+      cy.keyboardInput(aboutGoodsOrServicesPage[DESCRIPTION].textarea(), application.EXPORT_CONTRACT[DESCRIPTION]);
+      yesRadio().input().click();
+
+      submitButton().click();
```

Comment:
is there a command for the above?

Reply:
Good shout, i've updated `completeAndSubmitAboutGoodsOrServicesForm` to have a `includeFinalDestination` flag. Therefore can use this here and declare the flag as false.